akd_core/src/types/node_label/mod.rs

```diff
@@ -230,6 +230,6 @@ impl NodeLabel {
         if other.get_prefix(self.get_len()) != *self {
             return Direction::None;
         }
-        Direction::Some(other.get_bit_at(self.get_len()) as usize)
+        Direction::try_from(other.get_bit_at(self.get_len())).unwrap()
```

Comment:
Filing an issue here to track and remove this .unwrap() call in a future PR

Reply:
So I thought about it, but other.get_bit_at only returns a `0` or `1` so it should always be safe (i.e. not panic). It was either this or thread the direction error all the way around everywhere this parent function was called. But yeah not opposed to removing unwraps, just means more error bubbling when it shouldn't ever happen.